libs/dal/src/dal.service.ts

```diff
@@ -1,16 +1,16 @@
 import mongoose, { Connection, ConnectOptions } from 'mongoose';
 
+export const baseConfig: ConnectOptions = {
+  autoIndex: process.env.MONGO_AUTO_CREATE_INDEXES === 'true',
+  maxIdleTimeMS: process.env.MONGO_MAX_IDLE_TIME_IN_MS ? Number(process.env.MONGO_MAX_IDLE_TIME_IN_MS) : 1000 * 60,
+  maxPoolSize: process.env.MONGO_MAX_POOL_SIZE ? Number(process.env.MONGO_MAX_POOL_SIZE) : 50,
```

Comment:
The same for the maxPoolSize default and maxIdleTimeMS, it will introduce a breaking change for the self-hosted users, i wonder if we should just change the MONGO_MAX_POOL_SIZE to 50 for us.

Reply:
Great callout! Fixed!